README.md

```diff
@@ -174,4 +174,178 @@ PS C:\Users\Prince BUGINGO\Desktop\Alec\Bundle1Exercise1>
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % touch team.html
+
+```
+
+## Adding content on the html files 
+
+```bash 
+
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % touch team.html
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % echo "<h1>Home page</h1>" > home.html 
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % echo "<h1>About us</h1>" > about.html
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % echo "<h1>our team</h1>" > team.html 
+
+```
+## Staging and stashing the files 
+
+```bash 
+
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % git status
+On branch dev
+Untracked files:
+  (use "git add <file>..." to include in what will be committed)
+        about.html
+        home.html
+        team.html
+
+nothing added to commit but untracked files present (use "git add" to track)
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % git add .
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % git stash push -m "Added content on the files"
+Saved working directory and index state On dev: Added content on the files
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % 
+
+```
+
+## pop and restore them all
+
+```bash
+
+On branch dev
+Changes to be committed:
+  (use "git restore --staged <file>..." to unstage)
+        new file:   about.html
+        new file:   home.html
+        new file:   team.html
+
+Dropped refs/stash@{0} (60f308fc0598cafb4967715efd9c3d549286179d)
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % 
+
+```
+
+## stashing the team.html file 
+
+```bash
+
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % git restore --staged about.html home.html 
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % git status
+On branch dev
+Changes to be committed:
+  (use "git restore --staged <file>..." to unstage)
+        new file:   team.html
+
+Untracked files:
+  (use "git add <file>..." to include in what will be committed)
+        about.html
+        home.html
+
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % git stash push -m "stash team.html"
+Saved working directory and index state On dev: stash team.html
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % git status
+On branch dev
+Untracked files:
+  (use "git add <file>..." to include in what will be committed)
+        about.html
+        home.html
+
+nothing added to commit but untracked files present (use "git add" to track)
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % 
+
+```
+
+## Commit current changes and push them 
+
+```bash 
+
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % git add .
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % git status
+On branch dev
+Changes to be committed:
+  (use "git restore --staged <file>..." to unstage)
+        new file:   about.html
+        new file:   home.html
+
+gymikirenga@Ikirengas-iMac Gym-Git-Exercise-Solutions % git commit -m "New files with content and README.md update"
```

Comment:
So far well done. I would suggest you also make sure you write correct commit message. Use this link to learn about it: https://www.conventionalcommits.org/en/v1.0.0/